skimage/segmentation/slic_superpixels.py

```diff
@@ -213,10 +213,9 @@ def slic(image, n_segments=100, compactness=10., max_num_iter=10, sigma=0,
       interpret them as 3D with the last dimension having length 3, use
       `channel_axis=-1`.
 
-    * `start_label` is introduced to handle the issue [4]_. The labels
-      indexing starting at 0 will be deprecated in future versions. If
-      `mask` is not `None` labels indexing starts at 1 and masked area
-      is set to 0.
+    * `start_label` is introduced to handle the issue [4]_. Label indexing
+      starts at 1 by default. The labels' index starting at 0 will be
+      deprecated in future versions.
```

Comment:
Isn't it already deprecated?
```suggestion
      starts at 1 by default.
```